docs/notebooks/formulary/helio/collisional_analysis.ipynb

```diff
@@ -0,0 +1,513 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Collisional Analysis\n",
+    "\n",
+    "[coal]: ../../../api_static/plasmapy.formulary.collisions.helio.collisional_analysis.rst\n",
+    "[maruca2013]: https://www.doi.org/10.1103/PhysRevLett.111.241101\n",
+    "\n",
+    "This notebook will introduce and demonstrate collisional analysis, the functionality is contained in the [collisional_analysis()][coal] module. Collisional analysis, first introduced by [Maruca 2013][maruca2013], uses the idea that Coulomb collisions — \"soft,\" small-angle deflections mediated by the electrostatic force, act to erode non-local thermal equilibrium (LTE) features within the solar wind. Though remnants may persist through much of the inner heliosphere and beyond, this results in strong departures from LTE.\n",
+    "\n",
+    "The theory behind this technique is detailed in the first section below, after which the functionality is demonstrated in the second section. Observations from WIND are then utilized in the final section to validate the technique.\n",
```

Comment:
```suggestion
    "The theory behind this technique is detailed in the first section below. The functionality is demonstrated in the second section. Observations from WIND are then utilized in the final section to validate the technique.\n",
```

...since I try to keep sentences short when possible.